src/qml/NightstandPage.qml

```diff
@@ -130,6 +136,15 @@ Item {
                     value: nightstandDelay.value
                     onValueChanged: nightstandDelay.value = value
                 }
+
+                LabeledSwitch {
+                    height: rowHeight
+                    width: parent.width
+                    //% "Always on Display"
+                    text: qsTrId("id-always-on-display")
+                    checked: nightstandAlwaysOnDisplay.value
+                    onCheckedChanged: nightstandAlwaysOnDisplay.value = checked
```

Comment:
See other PR. Adding `displaySettings.lowPowerModeEnabled = checked` here might solve the issue as well.

Reply:
Good point -- fixed and also changed so that the AOD setting defaults to "on" as suggested in the other PR.